src/wizard/long.scss

```diff
@@ -5,9 +5,13 @@
 @mixin iui-wizards-long {
   @include iui-wizards;
 
-  > .iui-wizards-wrapper > .iui-wizards-step > .iui-wizards-step-title {
-    display: none;
-  }
+  > .iui-wizards-wrapper > .iui-wizards-step {
+    width: auto;
```

Comment:
Is this the magical fix? 😄

Reply:
It changes the demo image slightly 🤔

Reply:
Yeah, at least for the long wizard.

For the default wizard we needed to set a fixed width (20%) because we can't know how long the step's text description might be.

For the long wizard the text description is not underneath the step, so we can reset the width back to auto.

Reply:
wait this is in long.scss. What does it give?

Reply:
oh it's to reset